### PR TITLE
Migrating to actix 4

### PR DESCRIPTION
**Pull request by augustocdias**
#### Checklist:

- [x] Tests for the changes have been added (_for bug fixes / features_);
- [x] Docs have been added / updated (_for bug fixes / features_).
- [x] This PR has been added to [CHANGELOG.md](https://github.com/DDtKey/actix-web-grants/blob/main/CHANGELOG.md) (to [Unreleased] section);

### CODE REVIEW

**Comment by augustocdias** [comment]
Please lets wait for the final version before merging.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - 2021-xx-xx
 ### Added
+- Update actix-web to 4.0
 
 ### Changed
 - Update `actix-rt` dev-dependency to `2`
```
Comment (DDtKey) [inline CHANGELOG.md:12]:
```suggestion
### Changed
- Update actix-web to `4.0`
- Update `actix-rt` dev-dependency to `2` 
```

File: Cargo.toml [hunk 2]
```diff
@@ -20,8 +20,8 @@ default = ["macro-check"]
 macro-check = ["actix-grants-proc-macro"]
 
 [dependencies]
-actix-web = { version = "3.0.0", default-features = false}
-actix-grants-proc-macro = { version = "1", optional = true }
+actix-web = { version = "4.0.0-beta.3", default-features = false }
+actix-grants-proc-macro = { path = "./proc-macro", optional = true }
```
Comment (DDtKey) [inline Cargo.toml:24]: The version is required for cases of crate publication.
The Cargo book suggests specifying both parameters in such cases.
Before release, we will just bump the version.

```suggestion
actix-grants-proc-macro = { path = "./proc-macro", version = "1", optional = true }
```

**Review by DDtKey — COMMENTED** [review]
Thanks a lot for your contribution! 👍 
I left a couple of small comments, please take a look 🙂

**Review by DDtKey — APPROVED** [review]

**Comment by DDtKey** [comment]
@augustocdias could you merge the changes from the `main` branch into the PR and update the` actix-web` to `4.0.0-beta.5`, please? 
I want to create a separate branch for version 4 and merge to `main` in the future after stabilization.

**Comment by augustocdias** [comment]
Hi @DDtKey. It is updated

**Comment by DDtKey** [comment]
Thanks again for your PR 👍 
You have done an amazing job 🚀

**Comment by DDtKey** [comment]
I published the beta version of `actix-web-grants`: 
https://crates.io/crates/actix-web-grants/3.0.0-beta.1

**Pull request merged**